### ISSUE TITLE
Problem on the converge of collision value

### ISSUE THREAD

**Issue by ftyghome** [peasant98/Dronet-Pytorch#3]
Hello, thanks for your great work on the PyTorch implementation of Dronet :D 
In the training progress, I actually come with some problems and wonder if it is my fault.
I downloaded the Udacity steering dataset and the collision dataset collected on bikes, by running the move_data.py and do some small tweaks to the structure of folders, I made the dronet_torch_train work. However, the training result is not ideal. The loss plot shows that the steering value converged at the end, while the training result of the collision data looks not so good.
![image](https://user-images.githubusercontent.com/22185413/118973779-a0e3f080-b9a4-11eb-8618-fba00aa82b3a.png)
I have been trying to resolve the issue for days, but I still not find what I did wrong. Could you please offer me some advice on it?
Thanks:)

The structure of the "data" folder is as follows:
![image](https://user-images.githubusercontent.com/22185413/118975063-26b46b80-b9a6-11eb-8cc5-d4f4316c7e79.png)
![image](https://user-images.githubusercontent.com/22185413/118975074-2b791f80-b9a6-11eb-87cd-0d85eb1c6bc1.png)
![image](https://user-images.githubusercontent.com/22185413/118975085-2fa53d00-b9a6-11eb-9d67-6e34e77451e1.png)

**Comment by peasant98** [comment]
Hi @ftyghome, I'll check out the problem today and for the rest of this week.

**Comment by peasant98** [comment]
My apologies for the very, very late reply -- I will 100% address this in the coming days!

**Comment by ftyghome** [comment]
At your convenience :) I am working on another project recently but it is really glad to see this still active lol

**Comment by daothanh2011** [comment]
Hello, I have the same problem, could you pls give me any hint to improve the accuracy of collision

**Comment by peasant98** [comment]
Hi @daothanh2011, I've been quite busy with other matters (new job, etc) for some time -- can you remind me again 1 week from now?
